package.json

```diff
@@ -11,15 +11,20 @@
     "start:dev": "npm run webpack:dev & nodemon server/index.js --ignore public --ignore src",
     "test": "node server/db/seed.js & jest",
     "test:watch": "jest --watchAll",
-    "lint": "eslint src",
-    "lint:fix": "eslint --fix src",
-    "prettier": "prettier --write --no-semi --single-quote \"src/**/*.js\""
+    "lint": "eslint src server",
+    "lint:fix": "eslint --fix src server",
+    "prettier": "prettier --write --single-quote \"src/**/*.js\" \"server/**/*.js\" "
```

Comment:
Did you intend to add a --ignore in this?

Reply:
Oh - i meant to take that out completely. it should just reference the rules defined lower in the file

Reply:
I think we just fix now before we say this is good to go then.

Reply:
give me a sec to remove from the file

Reply:
sorry i read your comment too quickly. I made a .ignoreprettier file which houses the tests. I figure this was easier especially if we decide to include other file or folder types

Reply:
Oh. I missed that one. It was literally right at the top of the files changed.